tests/main/views/test_marketplace.py

```diff
@@ -551,10 +586,6 @@ def test_supplier_start_application(self):
 
     def test_supplier_applied_view_application_for_live_opportunity(self):
         self.login_as_supplier()
-        # mocking that we have applied
```

Comment:
yep something like `# this supplier has applied for this brief as part of our brief responses fixture in the setup`